### ISSUE TITLE
An Undersea Photo

### ISSUE THREAD

**Issue by danrabbit** [elementary/wallpapers#132]
Cassidy proposed a jellyfish shot, but it wasn't quite the right stuff. An undersea photo would be cool though, so noting it here in case we wanna gather ideas

**Comment by GammaGames** [comment]
How about some coral?
* https://unsplash.com/photos/7XjqE50Upak (A bit busy)
* https://unsplash.com/photos/P3qllUbhOII (Maybe too dark)
* https://unsplash.com/photos/41Wuv1xsmGM (I like the composition but might be too grainy)
* https://unsplash.com/photos/8QWH-42r1SY (I like the colors)

**Comment by cassidyjames** [comment]
@GammaGames I'm not opposed to coral, but those are all really grainy/fuzzy to me.

**Comment by GammaGames** [comment]
Yeah, I wasn't able to find many clear underwater photos on unsplash so I was intending to use them more as possible directions to go when searching than suggestions to include.

**Comment by martinstork** [comment]
My (coral) suggestion: https://www.pexels.com/photo/coral-3204592

**Comment by SubhadeepJasu** [comment]
Here's a couple I liked:
https://unsplash.com/photos/luanaxyvbpQ
https://unsplash.com/photos/KLbUohEjb04
https://unsplash.com/photos/IRsY8g_TvxM
https://unsplash.com/photos/ekthrVC_DVs
https://unsplash.com/photos/cl1HHDy8ZoA
https://unsplash.com/photos/xAwtDLOTB2A
